### ISSUE TITLE
Flash BPI-CM4 modules

### ISSUE THREAD

**Issue by nerocide** [turing-machines/turing-pi#47]
Same as for RK1 and RPI-CM4 It would be nice to have this feature to upload new img to `banana pi CM4`'s EMMC.

https://wiki.banana-pi.org/Banana_Pi_BPI-CM4

This might be possible with:
https://wiki.banana-pi.org/Getting_Started_with_CM4#Install_Image_with_Aml_Flash_Tool

**Comment by svenrademakers** [comment]
Can be picked up after #22

**Comment by ruslashev** [comment]
A problem I see here is that BPI-CM4 is not entirely pin-compatible: pin 93, where RPI-CM4 has nRPIBOOT, is not connected and it requires physically shorting pins, unlike what we programmatically do for RPI-CM4. So perhaps it is possible, but with a slight nuisance during flashing.
